### PR TITLE
onboard sUSDe - Review

### PR DESCRIPTION
**Pull request by marczeller**
funding.json is for a retroPGF optimism grant it's not linked to current AIP but bundling it up

### CODE REVIEW

File: src/20240621_AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum/AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum_20240621.sol [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum, AaveV3EthereumEModes} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveV3PayloadEthereum} from 'aave-helpers/v3-config-engine/AaveV3PayloadEthereum.sol';
+import {EngineFlags} from 'aave-helpers/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-helpers/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {IV3RateStrategyFactory} from 'aave-helpers/v3-config-engine/IV3RateStrategyFactory.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+/**
```
Comment (sakulstra) [inline src/20240621_AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum/AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum_20240621.sol:11]:
```suggestion

/**
```

File: src/20240621_AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum/AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum_20240621.sol [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    listings[0] = IAaveV3ConfigEngine.Listing({
+      asset: sUSDe,
+      assetSymbol: 'sUSDe',
+      priceFeed: 0xb37aE8aBa6C0C1Bf2c509fc06E11aa4AF29B665A,
+      eModeCategory: AaveV3EthereumEModes.NONE,
+      enabledToBorrow: EngineFlags.DISABLED,
+      stableRateModeEnabled: EngineFlags.DISABLED,
+      borrowableInIsolation: EngineFlags.DISABLED,
+      withSiloedBorrowing: EngineFlags.DISABLED,
+      flashloanable: EngineFlags.DISABLED,
```
Comment (sakulstra) [inline src/20240621_AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum/AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum_20240621.sol:40]: why not?

File: src/20240621_AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum/OnboardSUSDeToAaveV3OnEthereum.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Onboard sUSDe to Aave V3 on Ethereum"
+author: "Aave Chan Initiative"
+discussions: "https://governance.aave.com/t/arfc-onboard-susde-to-aave-v3-on-ethereum/17191"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x9c6dcf2af7d79467f34e52b505c25d5e4c89af77f00396307d42fc4816797cd9"
+---
+
+## Simple Summary
+
+This is an ARFC to seek approval for the addition of sUSDe to Aave V3 on Ethereum, following the consensus reached in the [TEMP CHECK ](https://snapshot.org/#/aave.eth/proposal/0x3263dbee34f5afe513a1a6c5bcd5207684cee03473e1b3d7616993dbd87ab881) and [ARFC](https://snapshot.org/#/aave.eth/proposal/0x9c6dcf2af7d79467f34e52b505c25d5e4c89af77f00396307d42fc4816797cd9) stage.
```
Comment (sakulstra) [inline src/20240621_AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum/OnboardSUSDeToAaveV3OnEthereum.md:10]:
```suggestion
This is an ARFC to seek approval for the addition of sUSDe to Aave V3 on Ethereum, following the consensus reached in the [TEMP CHECK](https://snapshot.org/#/aave.eth/proposal/0x3263dbee34f5afe513a1a6c5bcd5207684cee03473e1b3d7616993dbd87ab881) and [ARFC](https://snapshot.org/#/aave.eth/proposal/0x9c6dcf2af7d79467f34e52b505c25d5e4c89af77f00396307d42fc4816797cd9) stage.
```

File: src/20240621_AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum/OnboardSUSDeToAaveV3OnEthereum.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+## Motivation
+
+As discussed in the TEMP CHECK, Ethena's synthetic dollar, USDe, provides a stable crypto-native solution for a cash and carry structured product. The staked version of USDe, sUSDe, earns yield from the protocol and has high potential for strong borrow demand.
```
Comment (sakulstra) [inline src/20240621_AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum/OnboardSUSDeToAaveV3OnEthereum.md:14]:
```suggestion
As discussed in the TEMP CHECK, Ethena's synthetic dollar, USDe, provides a stable crypto-native solution for a cash and carry structured product. The staked version of USDe, sUSDe, earns yield from the protocol and has high potential for strong collateral demand.
```

should be collateral right, as it cannot be borrowed

File: src/20240621_AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum/AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum_20240621.sol (hunk 2 above)
Comment (marczeller) [reply src/20240621_AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum/AaveV3Ethereum_OnboardSUSDeToAaveV3OnEthereum_20240621.sol:41]: not allowed to be borrowed so seems coherent, not really to die on this hill if you want to have it activated

**Review by sendra — APPROVED** [review]

**Review by sakulstra — APPROVED** [review]

**Pull request merged**